### PR TITLE
Add Door Lock support. r=gmarty

### PR DESCRIPTION
**Pull request by azasypkin**

### CODE REVIEW

File: app/js/views/services-list-item.jsx [hunk 1]
```diff
@@ -222,11 +222,21 @@ export default class ServicesListItem extends React.Component {
       case 'ip-camera@link.mozilla.org':
         return this.renderGenericService('Camera', 'ip-camera');
       case 'OpenZwave Adapter':
+        if (this.hasDoorLockChannel(this.props.getters) ||
```
Comment (azasypkin) [inline app/js/views/services-list-item.jsx]: There is no other way to distinguish them for now :/

**Comment by gmarty** [comment]
That looks good to me! r+

**Comment by azasypkin** [comment]
> That looks good to me! r+

Thanks! Landing then, fabrice also managed to check it on real lock - he didn't report any issue so far.

**Comment by JohanLorenzo** [comment]
I checked with the lock in Paris. I managed to create recipes that lock or unlock the door. One piece of trivia, you can create a rule that locks the door when foxbox detects it's unlocked.

**Comment by azasypkin** [comment]
> I checked with the lock in Paris. I managed to create recipes that lock or unlock the door. It correctly detected even if the Z-wave dongle deals with another sensor (like the motion one)

Yay, thanks for verifying! Then clearly I have issue with the emulator - will investigate.

**Pull request merged**